src/main/webapp/app/assessment/assessment-locks/assessment-locks.component.html

```diff
@@ -57,7 +57,11 @@ <h2>
                             </button>
                         </span>
                         <span>
-                            <button *ngIf="!submission.result.completionDate" (click)="cancelAssessment(submission)" class="btn btn-outline-secondary btn-sm mb-1">
+                            <button
+                                *ngIf="!getLatestSubmissionResult(submission)!.completionDate"
```

Comment:
@fde312 mentioned below that it might be better to refactor that into a variable so that we avoid long html